charts/apisix-dashboard/templates/configmap.yaml

```diff
@@ -65,3 +65,7 @@ data:
           password: {{ .password }}
         {{- end }}
     {{- end }}
+    plugins:
+      {{- range .Values.config.conf.plugins }}
```

Comment:
We should check the plugins list and generate this field only when it is non-empty

Reply:
This is a great call ✅